### ISSUE TITLE
[3rd] Tradução Cap. 21 - "Project: Skill-Sharing Website"

### ISSUE THREAD

**Issue by joaostein** [braziljs/eloquente-javascript#420]
Responsável: @chrqq

**Comment by joaostein** [comment]
@chrqq, estou enviando essa mensagem para todos os responsáveis apenas para checar se está tudo certo com a sua tradução.  Se precisar de qualquer coisa, só avisar.

_ps. A data da [milestone](https://github.com/braziljs/eloquente-javascript/milestone/2) para finalizar a tradução está para 24 de Dezembro (ho-ho 🎅🎁).  Se você achar que não for realista, só avisar, ok?_

**Comment by joaostein** [comment]
@chrqq - poderia dar um update e qual o seu plano para tradução deste capítulo?

**Comment by Leo3965** [comment]
Olá @joaostein, posso realizar a tradução desse capítulo?

**Comment by joaostein** [comment]
@Leo3965 claro, atribuí o capítulo para você. Se tiver qualquer dúvida é só me avisar.

**Comment by Leo3965** [comment]
@joaostein tradução iniciada

**Comment by joaostein** [comment]
Encerrando — pulamos a 3ª edição e fomos direto pra 4ª (#490).
